fedramp-integrated-inventory-workbook/deployment/inventory/mappers.py

```diff
@@ -0,0 +1,262 @@
+        ip_addresses: List[str] = []
+
+        for availabilityZone in availabilityZones:
+            if load_balancer_addresses := availabilityZone.get("loadBalancerAddresses"):
+                for load_balancer_address in (load_balancer_address for load_balancer_address in load_balancer_addresses if
+                                              "ipAddress" in load_balancer_address):
+                    ip_addresses.append(load_balancer_address["ipAddress"])
+
+        return ip_addresses
+
+    def _do_mapping(self, config_resource: dict) -> List[InventoryData]:
+        data_list: List[InventoryData] = []
+
+        data = {"asset_type": self._get_asset_type_name(config_resource),
+                "unique_id": config_resource["arn"],
+                "is_virtual": "Yes",
+                "software_vendor": "AWS",
+                "is_public": "Yes" if config_resource["configuration"]["scheme"] == "internet-facing" else "No",
+                # Classic ELBs have key of "vpcid" while V2 ELBs have key of "vpcId"
+                "network_id": config_resource["configuration"]["vpcId"] if "vpcId" in config_resource["configuration"] else
+                config_resource["configuration"]["vpcid"],
+                "owner": _get_tag_value(config_resource["tags"], "owner")}
+
+        if len(ip_addresses := self._get_ip_addresses(config_resource["configuration"]["availabilityZones"])) > 0:
+            for ip_address in ip_addresses:
+                data = copy.deepcopy(data)
+
+                data["ip_address"] = ip_address
+
+                data_list.append(InventoryData(**data))
+        else:
+            data_list.append(InventoryData(**data))
+
+        return data_list
+
+
+class RdsDataMapper(DataMapper):
+    def _get_supported_resource_type(self) -> List[str]:
+        return ["AWS::RDS::DBInstance"]
+
+    def _do_mapping(self, config_resource: dict) -> List[InventoryData]:
+        data = {"asset_type": "RDS",
+                "unique_id": config_resource["arn"],
+                "is_virtual": "Yes",
+                "software_vendor": "AWS",
+                "authenticated_scan_planned": "No",
+                "purpose": self.REQUIRES_MANUAL_INPUT,
+                "is_public": "Yes" if config_resource["configuration"]["publiclyAccessible"] else "No",
+                "hardware_model": config_resource["configuration"]["dBInstanceClass"],
+                "software_product_name": f"{config_resource['configuration']['engine']}-{config_resource['configuration']['engineVersion']}",
+                "network_id": config_resource['configuration']['dBSubnetGroup']['vpcId'] if "dBSubnetGroup" in config_resource[
+                    'configuration'] else '',
+                "owner": _get_tag_value(config_resource["tags"], "owner"),
+                "location": config_resource["awsRegion"]}
+
+        return [InventoryData(**data)]
+
+
+class DynamoDbTableDataMapper(DataMapper):
```

Comment:
Doesn't hurt to leave in mappers we don't have but we don't have any plans to use Dynamo for anything.